deps/chakrashim/src/inspector/v8-debugger.cc

```diff
@@ -77,22 +77,25 @@ void V8Debugger::getCompiledScripts(
     int contextGroupId,
     std::vector<std::unique_ptr<V8DebuggerScript>>& result) {
   JsValueRef scripts;
-  JsDiagGetScripts(&scripts);
+  if (JsDiagGetScripts(&scripts) != JsNoError) {
+    assert(false);
```

Comment:
@kfarnung  - curious, is this failing any unit test? I am doing a merge and [this](https://github.com/nodejs/node/blob/master/test/inspector/test-bindings.js) test case is failing because of this issue. I will ignore this failure for now.